hoot-core/src/main/cpp/hoot/core/elements/OsmUtils.cpp

```diff
@@ -737,4 +746,221 @@ bool OsmUtils::anyElementsHaveAnyKvp(const QStringList& kvps,
   return anyElementsHaveAnyKvp(kvps, elements);
 }
 
+std::shared_ptr<geos::geom::Geometry> OsmUtils::_getGeometry(
```

Comment:
So, these were brought over directly from `PoiPolygonInfoCache`. I thought I needed them for this PR but that didn't end up being so. However, I think having these here will be more useful in the future. In `PoiPolygonInfoCache` most of these are backed by a cache for performance improvements but these versions aren't. Opened #3719 to clean the two classes up and get rid of duplicate code.